common/src/logger/loggers/mod.rs

```diff
@@ -165,6 +184,9 @@ pub fn create_root_logger() -> Logger {
 
 /// Create a logger that is suitable for use during test execution.
 pub fn create_test_logger(test_name: String) -> Logger {
+    if env::var("MC_LOG_STDERR").is_err() {
```

Comment:
This means there's no way to disable this for tests, since the check above doesn't compare the value of `MC_LOG_STDERR`, just its existence.

Reply:
that's my intention, tests should always log to stderr

Reply:
Ah, I see. A comment explaining that would be helpful. Thanks!